dev/drivers/scripts/stats/wafs/jevs_wafs_atmos_stats.sh

```diff
@@ -37,7 +37,7 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 # environment variables set
 ############################################################
 export envir=prod
-export COMINgfs=/lfs/h1/ops/para/com/gfs/${gfs_ver}
+export COMINgfs=/lfs/h1/ops/prod/com/gfs/${gfs_ver}
```

Comment:
To check that incoming WAFS implementation won't break EVS because of pressure level updates, please modify 'COMINgfs' with 'para': export COMINgfs=/lfs/h1/ops/para/com/gfs/${gfs_ver}

Both 'prod' and 'para' of 'COMINgfs' work.

Reply:
@YaliMao-NOAA Which will be the version that you want to be updated in the code?  With COMINgfs set to prod or to para?  

To be complete, I'll run tests for both.